package.json

```
@@ -4,13 +4,14 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
+    "precibuild": "npm i gatsby-plugin-sharp gatsby-transformer-sharp gatsby-remark-images gatsby-plugin-manifest && sed -i 's/\\/\\/ CI BUILD //g' gatsby-config.js",
```

i'm afraid these packages can no longer being version controlled by npm and the build could be broken due to any breaking changes in these packages.
it does not promise builds built in different times to be working consistently, and may fail silently **in build** without developing environment notice

---

这个确实是没办法的事情，考虑以后这个变成主站，让每一个人都能顺利的编译&&下载好 sharp 是不太现实的，至少在本项目的 contrib 中，还有不少人非常受这个东西困扰。

不知道有没有更好的解决方案

---

`optionalDependencies` may help
and who may have problems in internet connection can simply `npm install --no-optional`

---

lgtm